src/platform/generic/egl/surface.rs

```diff
@@ -93,39 +93,36 @@ impl EGLBackedSurface {
 
         unsafe {
             // Create our texture.
-            let mut texture_object = 0;
-            gl.GenTextures(1, &mut texture_object);
+            let texture_object = gl.create_texture().unwrap();
```

Comment:
It would be nice to have error handling one day.

Reply:
Actually to be compatible with old code it should be `.ok()` instead of unwrap.